projects/manager/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": ["projects/manager/tsconfig.app.json", "projects/manager/tsconfig.spec.json"],
+                "createDefaultProgram": true
+            },
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
+    ]
+}
```

Comment:
Maybe we should have a default filter on datasets that are owned by the user?
What is the purpose for this console other than to manage your stuff?

Reply:
yeah makes total sense, but the API doesn't yet support this, so it's not that easy. Maybe a first version could go without it?

If not I will implement it.

Reply:
we could make a list by priority and start with the most pressing

Reply:
I started collecting tasks here https://docs.google.com/document/d/1snSQWNaQuLdXq3PBRk-T-bhSkY5Y3tdsDkf5s14mtsI/edit